com.unity.shadergraph/Editor/Generation/Collections/IncludeCollection.cs

```diff
@@ -1,61 +1,85 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
+using UnityEngine;
 
 namespace UnityEditor.ShaderGraph
 {
     [GenerationAPI]
-    internal class IncludeCollection : IEnumerable<IncludeCollection.Item>
+    [Serializable]
```

Comment:
The changes here are to make the IncludeCollection serializable, so we can serialize it as part of the subgraph asset.

Reply:
Note that I removed "Item" and just combined it with the IncludeDescriptor.  No one else was using IncludeDescriptor, so there didn't seem to be any reason to have separate classes for the two.